pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -273,7 +276,8 @@ func ReadLogs(ctx context.Context, path, containerID string, opts *LogOptions, r
 		return fmt.Errorf("failed to open log file %q: %v", path, err)
 	}
 	defer f.Close()
-
+	b, _ := ioutil.ReadFile(path)
+	klog.Info("COMPLETE   ===========================   FILE   DATA  =================  ", string(b))
```

Comment:
This feels like a very big change - we're logging the full logs?

Reply:
Yeah, only it's failing only in GKE. 
Actually gke is writing the container termination status message with `\n`.
None of the other environment writing like this. Please check kubelet log. Search DONE\n